src/openrct2/drawing/Sprite.cpp

```diff
@@ -36,6 +36,75 @@
 using namespace OpenRCT2;
 using namespace OpenRCT2::Ui;
 
+static inline uint32 rctc_to_rct2_index(uint32 image) {
+    if      (image < 1575                   ) return image;
```

Comment:
Could you align the operator and number on this line with those below? It makes for easier reading.

Normally I wouldn't like single-line if block, but on this occasion, it's all right.